I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/story/Prologue.rpy", line 347, in script
    show bg communal_room with screen_shake
Exception: Parameter 'old_widget' is not known by ATL Transform.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/story/Prologue.rpy", line 347, in script
    show bg communal_room with screen_shake
  File "C:\Users\Troy\renpy-8.2.3-sdk\renpy\ast.py", line 1363, in execute
    renpy.exports.with_statement(trans, paired=paired)
  File "C:\Users\Troy\renpy-8.2.3-sdk\renpy\exports.py", line 1782, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\Troy\renpy-8.2.3-sdk\renpy\display\core.py", line 1549, in do_with
    return self.interact(trans_pause=True,
  File "C:\Users\Troy\renpy-8.2.3-sdk\renpy\display\core.py", line 2166, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\Troy\renpy-8.2.3-sdk\renpy\display\core.py", line 2609, in interact_core
    trans = instantiate_transition(None, old_root, layers_root)
  File "C:\Users\Troy\renpy-8.2.3-sdk\renpy\display\core.py", line 2519, in instantiate_transition
    trans = self.ongoing_transition[layer](
  File "C:\Users\Troy\renpy-8.2.3-sdk\renpy\atl.py", line 617, in __call__
    raise Exception('Parameter %r is not known by ATL Transform.' % k)
Exception: Parameter 'old_widget' is not known by ATL Transform.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.3.24061702
VN early_version 1.0
Mon Apr 14 16:52:00 2025
